run-build.sh

```diff
@@ -93,6 +96,14 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --linux-portable)
+            # Pass in a specific version (the one we set up to bootstrap this experience). Once 
+            # we have an official build we can remove the version parameter and use the latest.
+            LINUX_PORTABLE_INSTALL_ARGS="--linux-portable --version 2.0.0-alpha-005151"
+            # Until we get test support for 2.0 we need to pass in the targets without test.
+            LINUX_PORTABLE_BUILD_ARGS="/p:Rid=\"linux-x64\" /p:OSName=\"linux\" /p:IncludeAdditionalSharedFrameworks=false /p:CLITargets=\"Prepare;Compile;Package;Publish\""
```

Comment:
I don't think `p:IncludeAdditionalSharedFrameworks` should be required to be passed in on the command line.  It should be calculated from the RID, same as what I did here:

https://github.com/dotnet/cli/commit/916a19c4936d92b92b5fbef2b801dba9c125d0db#diff-d595ac18a42f088f16eefcd37b417b04R136